lineups/nosalive_2022.txt

```diff
@@ -0,0 +1,47 @@
+Da Chick,1
+Lefty,1
+Andrew Cushin,1
+Acid Arab Live,1

```

Comment:
Can you drop the 'live' from this please? Spotify's search fails when lineups do this

```suggestion
Acid Arab,1
```